### ISSUE TITLE
Confusion metric bugs

### ISSUE THREAD

**Issue by JamesLe01** [J-A-T-T/LVO-EEG#1]
def evaluation_metric(CM):
    TP = CM[0][0]
    FP = CM[0][1]
    FN = CM[1][0]
    TN = CM[1][1]
    expected_loss = (4*FN+FP)/(4*TP+TN)
    return expected_loss

This is wrong.
Instead, it should be this according to sklearn documentation:

def evaluation_metric(CM):
    TN = CM[0][0]
    FP = CM[0][1]
    FN = CM[1][0]
    TP = CM[1][1]
    expected_loss = (4*FN+FP)/(4*TP+TN)
    return expected_loss
